sensor/common/scan/scan_test.go

```diff
@@ -689,7 +686,8 @@ func successfulScan(_ context.Context, _ *storage.Image,
 }
 
 func successfulFetchSignatures(_ context.Context, _ signatures.SignatureFetcher, _ *storage.Image, _ string,
-	_ registryTypes.Registry) ([]*storage.Signature, error) {
+	_ registryTypes.Registry,
+) ([]*storage.Signature, error) {
```

Comment:
Is it some kind of formatter, that did that? Looks unnecessary.

Reply:
I was curious about this as well and tend to agree on unnecessary.

Reply:
It's `gofumpt`, sometimes it does a bit too much.